ci/sync-mirrors.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+source "${BUILD_DIR}/ci/common/common.sh"
+source "${BUILD_DIR}/ci/common/dependencies.sh"
+
+WORK_DIR=${WORK_DIR:-${BUILD_DIR}/build}
+MIRROR_USER=${MIRROR_USER:-neovim}
+MIRROR_BRANCH=${MIRROR_BRANCH:-master}
+
+sync_bzr_mirror() {
+  local repo="${1}"
+  local mirror_url="git://github.com/${MIRROR_USER}/${repo}.git"
+  local upstream_url="http://bazaar.leonerd.org.uk/c/${repo}"
+  local repo_dir="${WORK_DIR}/${repo}"
+
+  echo "Cloning Git mirror repo ${mirror_url}:${MIRROR_BRANCH} into ${repo_dir}."
+  rm -rf "${repo_dir}"
+  git clone --branch "${MIRROR_BRANCH}" --depth 1 "${mirror_url}" "${repo_dir}"
+
+  (
+    cd "${repo_dir}"
+    echo "Importing ${upstream_url} into Git branch ${repo}-import."
+    git bzr import "${upstream_url}" "${repo}-import"
+
+    echo "Fast-forwarding master to ${repo}-import."
+    git merge --ff-only "${repo}-import"
```

Comment:
Maybe the branch should be named just `master-upstream`. In cmake scripts this might be more intuitive than see "libvterm-import" for example.

Or we should just overwrite master. Any changes we make to our mirrors should be on non-master branches.